lib/backgrounder/workers/process_asset_mixin.rb

```diff
@@ -16,23 +14,22 @@ def perform(*args)
 
         return unless record && asset_present?(asset)
 
-        recreate_asset_versions!(asset)
+        process_asset_by_cache!(asset)
+
+        return unless record.respond_to?(:"#{column}_processing")
```

Comment:
There will be some unrelated Rubocop changes in here because of my setup, so sorry about that. The main changes in this PR are to use `cache!` instead of `recreate_versions!`